# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/VideoConduit.h
# Commit: e0a2e54e4de8
# Full Hash: e0a2e54e4de80813829cbcf9e4a5e4ff1285d451
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 9
# Description:
#   Bug 1654112 - Move WebRtcCallWrapper to dedicated files. r=ng
#   
#   This patch also renames it WebrtcCallWrapper, to better match neighboring
#   classes starting on "Webrtc".
#   
# ==============================================================================

diff -r 1965757ee924 -r e0a2e54e4de8 dom/media/webrtc/libwebrtcglue/VideoConduit.h
--- a/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Thu Sep 02 13:22:03 2021 -0500
+++ b/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Thu Apr 08 11:31:43 2021 +0200
@@ -98,7 +98,7 @@
    */
   void ReceivedRTCPPacket(const uint8_t* data, int len) override;
   Maybe<DOMHighResTimeStamp> LastRtcpReceived() const override;
-  DOMHighResTimeStamp GetNow() const override { return mCall->GetNow(); }
+  DOMHighResTimeStamp GetNow() const override;
 
   MediaConduitErrorCode StopTransmitting() override;
   MediaConduitErrorCode StartTransmitting() override;
@@ -220,7 +220,7 @@
     return mCodecMode;
   }
 
-  WebrtcVideoConduit(RefPtr<WebRtcCallWrapper> aCall,
+  WebrtcVideoConduit(RefPtr<WebrtcCallWrapper> aCall,
                      nsCOMPtr<nsISerialEventTarget> aStsThread,
                      std::string aPCHandle);
   virtual ~WebrtcVideoConduit();
@@ -418,7 +418,7 @@
   // WEBRTC.ORG Call API
   // Const so can be accessed on any thread. Most methods are called on
   // main thread, though Receiver() is called on STS. This seems fine.
-  const RefPtr<WebRtcCallWrapper> mCall;
+  const RefPtr<WebrtcCallWrapper> mCall;
 
   // Written only on main thread. Guarded by mMutex, except for reads on main.
   webrtc::VideoSendStream::Config mSendStreamConfig;